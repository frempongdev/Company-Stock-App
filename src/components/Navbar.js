import React from 'react';
import '../styles/Navbar.css';

const Navbar = () => (
  <div className="nav-main">
    <h1>Company Stocks App</h1>
  </div>
);

export default Navbar;
