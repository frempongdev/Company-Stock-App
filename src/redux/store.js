import { configureStore } from '@reduxjs/toolkit';
import companiesReducer from './Companies/CompaniesSlice';

const store = configureStore({
  reducer: {
    companies: companiesReducer,
  },
});

export default store;
