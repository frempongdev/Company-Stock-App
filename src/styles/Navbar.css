.nav-main {
    /* border: solid rgb(0, 0, 0); */
    box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.5);
    font-family: 'Lato', sans-serif;
}

h1 {
    margin: 0%;
    padding: 2% 0%;
}