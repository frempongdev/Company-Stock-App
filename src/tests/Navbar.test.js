import { Provider } from 'react-redux';
import '@testing-library/jest-dom';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';
import store from '../redux/store';

describe('Navbar', () => {
  it('renders Navbar', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <BrowserRouter>
            <Navbar />
          </BrowserRouter>
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
