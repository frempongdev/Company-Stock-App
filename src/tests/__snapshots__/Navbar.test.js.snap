// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders Navbar 1`] = `
<div
  className="nav-main"
>
  <h1>
    Company Stocks App
  </h1>
</div>
`;
